ext/GraphIODOTExt.jl

```diff
@@ -0,0 +1,92 @@
+module GraphIODOTExt
+
+using Graphs
+import Graphs: loadgraph, loadgraphs, savegraph
+
+@static if isdefined(Base, :get_extension)
+    using GraphIO
+    using ParserCombinator
+    import GraphIO.DOT.DOTFormat
+else
+    using ..GraphIO
+    using ..ParserCombinator
+    import ..GraphIO.DOT.DOTFormat
+end
```

Comment:
Maybe we can add a `TODO` that we don't require this after we bump the min Julia version to `v1.9`?

Reply:
ok !